src/app/public/modules/image/image.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
```

Comment:
Import statements need to follow code style guidelines.
See: https://docs.blackbaud.com/skyux-team-docs/style#imports